ledger/participant-integration-api/src/main/scala/platform/index/IndexServiceOwner.scala

```diff
@@ -78,11 +78,19 @@ final class IndexServiceOwner(
         eventProcessingParallelism = config.eventsProcessingParallelism,
       )(servicesExecutionContext)
 
+      bufferedCommandCompletionsReader = BufferedCommandCompletionsReader(
+        transactionsBuffer = inMemoryState.transactionsBuffer,
+        delegate = ledgerDao.completions,
+        metrics = metrics,
+        eventProcessingParallelism = 1,
```

Comment:
Why 1? Should this get exposed in the config?

Reply:
This was an oversight, thanks. It is actually meant to be hard-coded (processing is a no-op) but it shouldn't be exposed to the IndexServiceOwner indeed.